File: 프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.py
def solution(price, money, count):
    # 놀이기구의 총 금액을 계산( n(n+1)/2 * 가격)
    total_price = price * ( (count*(count+1))/2 )
    return 0 if money >= total_price else total_price - money

File: 프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.java
class Solution {
    public long solution(int price, int money, int count) {
        /*
        count 까지의 시그마 구하기
        s = count*(count + 1)/2
        total = s * price
        */
        long s = count*(count+1)/2;
        long need = s*price;
        return need <= money ? 0 : need - money; 
    }
}